modules/tinymce/src/plugins/lists/main/ts/ui/Dialog.ts

```diff
@@ -47,10 +52,15 @@ const open = (editor: Editor) => {
     ],
     onSubmit: (api) => {
       const data = api.getData();
-      editor.execCommand('mceListUpdate', false, {
-        attrs: {
-          start: data.start === '1' ? '' : data.start
-        }
+      parseStartValue(data.start).each((detail) => {
```

Comment:
This is the behaviour change, previously an unknown list start was used verbatim. Now it's blocked.

We don't want to post invalid list start values, but we _do_ want to post empty strings. That's remove attribute (this is also what happens in the `detail.start === '1' ? '' : detail.start` code).